libs/features/mas/web-components/src/merch-card.js

```diff
@@ -716,6 +297,28 @@ export class MerchCard extends LitElement {
         );
     }
 
+    toggleActionMenu(e) {
```

Comment:
This method is only applicable to `catalog` card. I wonder if it should be moved to `catalog.js`?

Reply:
had some issues there but you are right, i'll try again :)